apps/cli/tf/bucket.tf

```diff
@@ -59,15 +59,13 @@ resource "aws_iam_policy" "sunodo_cli_s3_policy" {
 }
 
 module "github-oidc" {
-  source  = "terraform-module/github-oidc-provider/aws"
-
   # https://github.com/terraform-module/terraform-aws-github-oidc-provider
-  version = "~> 2.1.0"
-  create_oidc_provider = true
-  create_oidc_role     = true
-
+  source                    = "terraform-module/github-oidc-provider/aws"
+  version                   = "~> 2.1.0"
+  oidc_provider_arn         = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:oidc-provider/token.actions.githubusercontent.com"
```

Comment:
If we want to use terraform cloud, we could use variables in this section too . like `var.oidc_arn`.